lib/helpers.js

```diff
@@ -141,35 +157,6 @@ exports.checkRelationName = function(modelDefinition, name) {
 };
 
 /**
- * Get the root command name based on the env var
- * @returns {string}
- */
-exports.getCommandName = function() {
```

Comment:
Sure I will move it to the original line. I extracted it into new file then realized that old file still needs it, so moved it back, but seems not to the original line lol.